sdks/python/apache_beam/testing/synthetic_pipeline_test.py

```diff
@@ -57,29 +57,39 @@ class SyntheticPipelineTest(unittest.TestCase):
 
   # pylint: disable=expression-not-assigned
 
-  def test_synthetic_step(self):
-    start = time.time()
+  def test_count_elements_after_synthetic_step(self):
     with beam.Pipeline() as p:
       pcoll = p | beam.Create(list(range(10))) | beam.ParDo(
           synthetic_pipeline.SyntheticStep(0, 0.5, 10))
```

Comment:
Also I just realized that `delay` is here no longer tested in this testcase, so I'm removing it from here